pygfx/utils/gallery_scraper.py

```diff
@@ -1,3 +1,5 @@
+import re
```

Comment:
why is the gallery_scraper.py file not in the `docs` folder actually? are we shipping it so dependents of pygfx can use it to build their own galleries?

Reply:
I first thought it was because Sphinx required it for some reason, but I think your explanation is correct. That way e.g. fastplotlib and others can create a gallery without replicating the scraping mechanics.